### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

File: apps/files/controller/apicontroller.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ public function updateFileTags($path, $tags = null) {
 	 *
 	 * @NoAdminRequired
 	 *
-	 * @param array|string $tagName tag name to filter by
+	 * @param string[] $tagName tag name to filter by
```
Comment (nickvergessen) [inline apps/files/controller/apicontroller.php]: This is not equivalent?

Also the method that is called with this value is different:
```php
	/**
	* ....
	* @param string $tag Tag id or name.
```

Comment (nickvergessen) [inline apps/files/controller/apicontroller.php]: So basically the `array|` should be removed, nothing more

File: apps/files/lib/activity.php [hunk 2]
```diff
@@ -160,7 +160,7 @@ public function translate($app, $text, $params, $stripPath, $highlightParams, $l
 	 * @param string $text
 	 * @param IL10N $l
 	 * @param array $params
-	 * @return bool|string
+	 * @return string|false
```
Comment (nickvergessen) [inline apps/files/lib/activity.php]: false is no data type? :-1:

File: apps/files_external/lib/config.php [hunk 3]
```diff
@@ -357,7 +358,7 @@ public static function dependencyMessage($backends) {
 	 * @param \OCP\IL10N $l
 	 * @param string $module
 	 * @param string $backend
-	 * @return string
+	 * @return OC_L10N_String
```
Comment (nickvergessen) [inline apps/files_external/lib/config.php]: should type hint the returns instead of adjusting the docs

File: apps/files/lib/activity.php (hunk 2 above)
Comment (DeepDiver1975) [inline apps/files/lib/activity.php]: no data type but a valid keyword according to php doc - http://www.phpdoc.org/docs/latest/guides/types.html#keywords

Comment (nickvergessen) [inline apps/files/lib/activity.php]: ah okay, I remember something to complain about false and replacing it with bool

**Comment by nickvergessen** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**